frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -1202,6 +1179,27 @@ export const surveyLogic = kea<surveyLogicType>([
                 return !!enabledFlags[FEATURE_FLAGS.SURVEYS_PARTIAL_RESPONSES]
             },
         ],
+        timestampFilter: [
+            (s) => [s.survey, s.dateRange],
+            (survey: Survey, dateRange: SurveyDateRange): string => {
+                if (!dateRange) {
+                    return `AND timestamp >= '${getSurveyStartDateForQuery(survey)}'
+                AND timestamp <= '${getSurveyEndDateForQuery(survey)}'`
+                }
+
+                const fromDateDayjs = dateStringToDayJs(dateRange.date_from)
+                const fromDate = fromDateDayjs
+                    ? fromDateDayjs.format('YYYY-MM-DD HH:mm:ss')
```

Comment:
format string can be a const (and used twice at least). DRY

Reply:
nice catch, will use the format i had already defined in the other file